src/dotnet-specflow/Fixer.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+using System.Xml.XPath;
+
+namespace SpecFlow.Dnx
+{
+    internal class Fixer
+    {
+        public readonly string SpecFlowExe;
+
+        private const string AppConfigSpecFlowSectionDefinitionType = "TechTalk.SpecFlow.Configuration.ConfigurationSectionHandler, TechTalk.SpecFlow";
+
+        private const string AppConfigSpecFlowSectionDefinition = @"	<configSections>
+		<section name=""specFlow"" type=""" + AppConfigSpecFlowSectionDefinitionType + @""" />
+	</configSections>";
+
+        private const string AppConfigSpecFlowSectionElement = "unitTestProvider";
+
+        // TODO: Allow specifying other unit test providers.
+        private const string AppConfigSpecFlowSection = @"	<specFlow>
+		<" + AppConfigSpecFlowSectionElement + @" name=""xUnit"" />
+	</specFlow>";
+
+        public Fixer()
+        {
+            // TODO: Allow for other DNX install locations.
+            var users = Directory.GetDirectories(@"C:\Users");
```

Comment:
There is no longer an `Environment.SpecialFolder.UserProfile` so this is hardcoded now.